main.py

```diff
@@ -0,0 +1,23 @@
+from llms.groq_backend import GroqLLM
+from pipeline.summarizer import summarize_podcast_full
+from utils.io_utils import load_text, save_text
+from utils.evaluation import evaluate_summary
+
+def main():
+    input_path = "s20e06-from-supply-chain-management-to-digital-warehousing-and-finops.md"
+    output_path = "summary_groq.md"
```

Comment:
Maybe this too in case of duplicates or the user needing a different name output.